recipe/meta.yaml

```diff
@@ -21,6 +21,7 @@ requirements:
     - python
     - cf_units
     - fiona
+    - gdal
```

Comment:
I recommend you to pin `gdal` to `2.1.*` to avoid headaches.
Also, `fiona` will install `gdal` for you, do you really need it here?